libnavigation-core/src/main/java/com/mapbox/navigation/core/history/model/HistoryWaypoint.kt

```diff
@@ -0,0 +1,46 @@
+package com.mapbox.navigation.core.history.model
+
+import com.mapbox.geojson.Point
+import com.mapbox.navigator.ActiveGuidanceOptions
+
+/**
+ * Public representation for [ActiveGuidanceOptions.waypoints].
```

Comment:
Let's document the values directly here instead of linking to a class that we are don't want developers interacting with.